servicemesh-install/topics/install.adoc

```diff
@@ -1,21 +1,227 @@
 [[installing-service-mesh]]
-= Installing {ProductName}
+= Installing the {ProductName}
+
+Installing the {ProductShortName} involves creating a custom resource definition file, then installing the operator to create and manage the custom resource.
+
+[[creating-custom-resource]]
+= Creating a custom resource file
+
+In order to deploy the {ProductShortName} control plane, you need to deploy a custom resource.  A _custom resource_ is an object that extends the Kubernetes API or allows you to introduce your own API into a project or a cluster.  You define a custom resource as a yaml file that defines the object. Then you use the yaml file to create the object. The complete example below contains all of the supported parameters and will deploy {ProductName} {ProductVersion} images based on Red Hat Enterprise Linux (RHEL).
+
+[TIP]
+====
+Deploying an istio-installation file that includes all of the parameters ensures that you have installed all of the Istio components required to complete the tutorials included in this document.
+====
+
+.Full example istio-installation.yaml
+```
+apiVersion: "istio.openshift.com/v1alpha1"
+kind: "Installation"
+metadata:
+  name: "istio-installation"
+spec:
+  deployment_type: openshift
+  istio:
+    authentication: false
+    community: false
+    prefix: openshift-istio-tech-preview/
+    version: 0.2.0
+  jaeger:
+    prefix: distributed-tracing-tech-preview/
+    version: 1.7.0
+    elasticsearch_memory: 1Gi
+  kiali:
+    username: username
+    password: password
+    prefix: kiali/
```

Comment:
This is the community prefix. You want the product prefix here (openshift-istio-tech-preview/)